index.d.ts

```diff
@@ -273,6 +284,12 @@ declare module '@react-pdf/renderer' {
       register: (options: {
         family: string;
         src: string;
+        fonts?: [font: {
```

Comment:
This is syntax error. Assuming you want array of fonts

```suggestion
        fonts?: {
          src: string;
          fontStyle: string;
          fontWeight: string | number;
          [key: string]: any;
        }[];
```

this is correct syntax

Reply:
Thanks. I’ll fix it asap